include/mbedtls/platform_util.h

```diff
@@ -55,6 +65,37 @@ extern "C" {
  */
 void mbedtls_platform_zeroize( void *buf, size_t len );
 
+#if defined(MBEDTLS_HAVE_TIME_DATE)
+/**
+ * \brief      Platform-specific implementation of gmtime_r()
+ *
+ *             The function is a thread-safe abstraction that behaves
```

Comment:
I'd at least put an asterisk after "thread-safe", that points to the explanation below, regarding the necessity of respecting of the critical section that we define. "Thread-safe" is an extremely strict term, and race conditions are difficult to detect and debug, so I think we may consider being extra careful about the multi-threading related wording here.